docs/authentication/configuration/session-options.mdx

```diff
@@ -21,52 +21,45 @@ Fortunately, with Clerk you have to ability to fully control the lifetime of you
 
 ### Inactivity timeout
 
-Denotes the duration after which a session will expire and the user will have to sign in again, if they haven't been active on your site. By default, this setting is disabled for all newly created instances. To enable it and set your desired value, in the Clerk Dashboard, go to **[Sessions](https://dashboard.clerk.com/last-active?path=sessions)**.
+Denotes the duration after which a session will expire and the user will have to sign in again, if they haven't been active on your site. A user is considered inactive when the application is closed, or when the app stops refreshing the token.
 
-<Images
-width={3024}
-height={1652}
-src="/docs/images/authentication/session-options/inactivity-timeout.jpg"
-alt="The 'Sessions' configuration section in the Clerk Dashboard with a red outline around the 'Inactivity timeout' section. This option is toggled on, revealing an input for customizing the time duration until inactivity timeout. A red arrow is pointing to the 'Apply Changes' button."
-/>
+By default, this setting is disabled for all newly created instances. To enable it and set your desired value:
+
+1. Navigate to the Clerk Dashboard.
```

Comment:
```suggestion
1.  Navigate to the Clerk Dashboard and select your application.```

Reply:
great minds think alike! I did this in the last PR review before seeing this :) I totally agree